processing/src/main/java/io/druid/query/search/search/LexicographicSearchSortSpec.java

```diff
@@ -63,6 +68,12 @@ public String toString()
 
   @Override
   public boolean equals(Object other) {
-    return (other instanceof LexicographicSearchSortSpec);
+    return this == other || other instanceof LexicographicSearchSortSpec;
```

Comment:
Do you mean IDE-generated one? My IDE(eclipse Mars) could not generate `equals()` automatically becasue these classes do not have non-static fields. I copied `equals()` from `LexicographicSearchSortSpec` which dates back to [](https://github.com/druid-io/druid/commit/4c8495662f672f146bd1f7eb64cd3bd3096a7aed#diff-e3e13007b63516c87e96b8cfc1ae8a40)